can_tools/scrapers/official/DC/dc_base.py

```diff
@@ -33,7 +33,7 @@ def fetch(self) -> requests.models.Response:
         """
 
         # get yesterday's date for finding most recent file
-        date = pd.to_datetime(self.execution_dt) - pd.Timedelta(days=2)
+        date = pd.to_datetime(self.execution_dt) - pd.Timedelta(days=1)
```

Comment:
Why did this change from 2 to 1? Why was it 2 to start?

`execution_dt` is measured in UTC time -- Would we rather be using the `self._retrieve_dt(tz="US/Eastern") - pd.Timedelta(days=1)`?

Reply:
As far as I'm aware it should always have been 1, as the files are uploaded on a one-day delay. I'm not sure why it was 2.

Reply:
:+1: